src/test/kotlin/org/onflow/flow/sdk/crypto/CryptoTest.kt

```diff
@@ -60,6 +63,13 @@ internal class CryptoTest {
         assertNotNull(signer)
     }
 
+    @Test
+    fun `Get signer for ECDSA_SECP256k1`() {
```

Comment:
While we are looking at the signer class I think it makes sense to improve the quality of these tests because they follow the same paradigm as the old hasher tests... will have a look at sanity checks Go SDK is currently using